src/utils/convertUnit.ts

```diff
@@ -0,0 +1,28 @@
+// Main unit: centimeter
+export enum LengthUnit {
+  MILLIMETER = 0.1,
+  CENTIMETER = 1,
+  INCH = 2.54,
+  FEET = 30.48,
+  METER = 100,
+}
+
+// Main unit: degree
+export enum AngleUnit {
+  DEGREE = 1,
+  RADIAN = 57.3,
+}
+
+type Unit = LengthUnit | AngleUnit;
+
+/**
+ * @param {number} value Number that will be converted
+ * @param {Unit} from Type of the given number
+ * @param {Unit} to Type of the returned number
+ * @return {number} Return converted number
+ */
+export default (value: number, from: Unit, to: Unit): number => {
+  const multiplier: number =
+    Math.round((from / to + Number.EPSILON) * 1000) / 1000;
+  return value * multiplier;
+};
```

Comment:
Can we also have syntactic sugar methods for `toSimulatorUnits` and `fromSimulatorUnits`? Simulator uses MKS (meters, kilograms, seconds) and radians. This way, if we ever change the underlying units of the simulator (unlikely, but never know), any caller that's already doing `to/fromSimulator` will automatically be fine.

Reply:
Also, this should probably throw an error if the units cannot be converted (e.g. length -> angle)